Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_tools.yml

```
@@ -743,3 +793,82 @@
         name: power-cell-slot-component-slot-name-default
         startingItem: PowerCellRobotTool
         locked: true
+
+- type: entity
+  name: borg plasma cutter
+  parent: WeaponPlasmaCutter
+  id: WeaponPlasmaCutterBorg
+  description: A mining tool that fires low-damage plasma bolts at a short range. This one is modified for cyborg use.
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)